examples/camera-app/linux/src/camera-device.cpp

```diff
@@ -49,6 +49,9 @@ CameraDevice::CameraDevice()
 
     // Set the CameraHALInterface in CameraAVStreamManager.
     mCameraAVStreamManager.SetCameraDeviceHAL(this);
+
+    // Set the CameraHALInterface in WebRTCProviderManager.
+    mWebRTCProviderManager.SetCameraDeviceHAL(this);
```

Comment:
The WebRTCProviderManager needs the Media-controller. It is better to use a setter to put a reference of the media controller in the WebRTCManager and use that to register the transport.